src/Security/UserClassBuilder.php

```diff
@@ -223,27 +223,33 @@ private function addGetPassword(ClassSourceManipulator $manipulator, UserClassCo
     private function addGetSalt(ClassSourceManipulator $manipulator, UserClassConfiguration $userClassConfig)
     {
         // add getSalt(): always empty
-        $builder = $manipulator->createMethodBuilder(
+        $getSaltMethodBuilder = $manipulator->createMethodBuilder(
             'getSalt',
-            null,
+            '?string',
             false,
             ['@see UserInterface']
         );
         if ($userClassConfig->hasPassword()) {
-            $builder->addStmt(
+            $getSaltMethodBuilder->addStmt(
                 $manipulator->createMethodLevelCommentNode(
                     'not needed when using the "bcrypt" algorithm in security.yaml'
```

Comment:
`not needed when using modern algorithms in security.yaml` + corresponding changes in fixtures

Reply:
Done, thanks